jobs/write_back_github/write_back_github.sh

```diff
@@ -1,20 +1,22 @@
 #!/bin/bash 
```

Comment:
glad to see this script become a standalone runable script.
previous , it strongly rely on some pre-condition like "./build-config" folder existing..etc
good